scripts/tests/archive-node-test.sh

```diff
@@ -3,11 +3,18 @@
 set -x
 # test archive node on known archive db
 
+# Don't prompt for answers during apt-get install
+export DEBIAN_FRONTEND=noninteractive
+
+apt-get update
+apt-get install -y procps
```

Comment:
It seems for some CI jobs, e.g. shellcheck, package are installed ahead of time. But here we're installing on test. Is there a general rule of thumb?